Bio/PDB/PDBIO.py

```diff
@@ -294,6 +286,92 @@ def _get_atom_line(
 
             return _PQR_ATOM_FORMAT_STRING % args
 
+    @staticmethod
```

Comment:
Any particular reason to make this a `@staticmethod`?

Reply:
There are potentially quite a few examples in older Biopython code where we could do this - https://docs.python.org/2/library/functions.html#staticmethod says ``staticmethod`` was added in Python 2.2 and as a decorator in Python 2.4